app/AndroidManifest.xml

```diff
@@ -395,7 +395,7 @@
 
         <receiver
             android:name="org.commcare.provider.ExternalApiReceiver"
-            android:exported="false">
+            android:exported="true">
```

Comment:
I think we talked about this earlier but we should add a ticket to put this receiver behind a permission in CC 2.54. As such it's likely to be flagged in security audits.

Reply:
Yes we did and I have a note to create that ticket.